wallabag/wallabag
# Add users management UI

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| Fixed tickets | https://github.com/wallabag/wallabag/issues/2116
| License       | MIT

Should fix https://github.com/wallabag/wallabag/issues/2116

- remove the “add a user” from the config page
- add a CRUD on user
- fix some missing translations (+ bad indentation)

Planned for 2.2 but if we got time to review for 2.1, it's fine too.

Listing:
![](https://cl.ly/3p3z262C1b0V/Image%202016-10-01%20at%205.32.53%20PM.png)

Editing:
![](https://cl.ly/3M030B3y2c1g/Image%202016-10-01%20at%205.33.21%20PM.png)

Create:
![](https://cl.ly/2N0c112c1t2E/Image%202016-10-01%20at%205.33.44%20PM.png)
